tools/pandapower_eval.py

```diff
@@ -0,0 +1,36 @@
+# In order to install PandaPower run 'pip install pandapower'
```

Comment:
You also need to install matplotlib with `pip install matplotlib`

Reply:
Hmm, I had the impression that it was a dependency of pandapower. I will include it to the instructions here.

Reply:
Yeah, that would be smart of pandapower,  but it wasnt installed automatically in my case.